docs/BUILD

```diff
@@ -4,98 +4,104 @@ load("@io_bazel_skydoc//stardoc:stardoc.bzl", "stardoc")
 # the --incompatible_remap_main_repo flag.
 stardoc(
     name = "build_test_docs",
-    input = "//rules:build_test.bzl",
     out = "build_test_doc_gen.md",
+    input = "//rules:build_test.bzl",
```

Comment:
please be sure to pull master and merge :)

Reply:
Done.